fearless/CoreLayer/OperationFactory/NFT/AlchemyNFTOperationFactory.swift

```diff
@@ -20,7 +20,10 @@ final class AlchemyNFTOperationFactory {
         let authorizedUrl = url.appendingPathComponent(ThirdPartyServicesApiKeysDebug.alchemyApiKey)
         let endpointUrl = authorizedUrl.appendingPathComponent("getContractsForOwner")
         var urlComponents = URLComponents(string: endpointUrl.absoluteString)
-        urlComponents?.queryItems = [URLQueryItem(name: "owner", value: address)]
+        urlComponents?.queryItems = [
+            URLQueryItem(name: "owner", value: address),
+            URLQueryItem(name: "withMetadata", value: "true")
```

Comment:
what if there's no metadata? maybe we can extract at least nft name ?

Reply:
It doesn't work like a filter. Here we inform whether we want to load metadata for NFT or not. If not, the download is faster, but we need the metadata. The default value here was already true, I just indicated it more explicitly, if in the future we want to change something it will be obvious where to do it.